lutris/util/timer.py

```diff
@@ -2,20 +2,29 @@
 
 
 class Timer:
+    """Simple Timer class to time code in seconds"""
 
-    def start_t(self):
-        self.start = datetime.datetime.now()
+    def __init__(self):
+        self._start = None
+        self._end = None
+        self.finished = False
 
-    def end_t(self):
-        self.end = datetime.datetime.now()
+    def start(self):
+        self._start = datetime.datetime.now()
+        self.finished = False
```

Comment:
You have to reset self._end to None otherwise accessing duration after a second call to Timer.start() will behave unexpectedly.

Reply:
nice catch ^^